lib/bolt/outputter/human.rb

```diff
@@ -286,6 +286,18 @@ def print_plans(plans, modulepath)
                         "details and parameters for a specific plan.")
       end
 
+      def print_topics(topics)
+        @stream.puts(topics.join("\n"))
```

Comment:
A header like "Available topics are:" (similar to what you get if you enter an invalid topic) might be helpful here.